servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -120,7 +121,7 @@ protected void init(URI uri) {
     init(api, httpClient);
   }
 
-  protected void init(NessieApiV1 api, HttpClient httpClient) {
+  protected void init(NessieApiV1 api, @Nullable HttpClient httpClient) {
```

Comment:
The gRPC tests pass null for the `httpClient`, so we need to skip tests that rely on the `httpClient`